Redesign UI of login and sign in page

Taiki-Choda opened the pull request:
I redesigned UI for the login page and the Sign up page,
update to a modern design with improved visual clarity.

- title with logo to welcome users to zulip
- input form with light colored background (not to use much borders)
- use card-like design for organization information
- left side with symbolic color

### Social login button
this design allow users to find a button which they want to use quickly.
- grid layout and simplification of social buttons
- not to repeat "Sign up with" or "Login with" in social login buttons

## Sign up --Before
<img src="https://user-images.githubusercontent.com/86971544/201964880-bf4edae6-9818-4670-b4bf-230ee77a0a61.jpg" width="50%">

## Log in --Before
<img src="https://user-images.githubusercontent.com/86971544/201965079-0cd8f417-d011-4c50-859e-b4b649b4f5a0.jpg" width="50%">

## Sign Up
<img src="https://user-images.githubusercontent.com/86971544/201964661-21c75e7e-dc74-48e0-9d1b-cc3065903e88.jpg" width="50%">

## Login
<img src="https://user-images.githubusercontent.com/86971544/201964680-77853db1-fb95-4fde-b3db-0441200bb5b7.jpg" width="50%">


- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [ ] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [ ] Each commit is a coherent idea.
- [ ] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [ ] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.

alya commented:
Thanks! Could you please post your proposed designs in the #design stream on chat.zulip.org for discussion?

Since it looks like you're changing the title area as well, it would be great to have a more complete screenshot of the current page (both in the PR description and when you post), as well as including the "before" versions of both the sign-up and login pages.

alya commented:
It would also be helpful to include some comments regarding the design, and what you're aiming to accomplish with the proposed changes.

Taiki-Choda committed:
Sign up and Login:improve UI to modern one with visual clarity

Add a logo above the page title to welcome users to zulip.

-Input form with light colored background (not to use much borders)
-use card like design for organization information
-left side with symbolic color

Simplification of social login buttons
This allows users to find a button which they want to use quickly.

Taiki-Choda commented:
Oh sorry, I mistakenly uploaded before finish writing.
Now I added informations and details!
